src/QMCTools/QMCFiniteSize/QMCFiniteSize.cpp

```diff
@@ -79,16 +79,15 @@ bool QMCFiniteSize::validateXML()
       if (a)
       {
         pushDocument((const char*)a);
-        processPWH(XmlDocStack.top()->getRoot());
+        processPWH(xml_doc_stack_.top()->getRoot());
         popDocument();
       }
     }
     else if (cname == "qmcsystem")
     {
       processPWH(cur);
     }
-    else
-    {}
+    else {}
```

Comment:
I think we can simply remove this line. Please take care of this in your next PR.